source/config.ts

```diff
@@ -0,0 +1,19 @@
+import { existsSync, writeFileSync, readFileSync } from 'fs-extra'
+import { ConfigDefault, createDefaultConfig } from './config.default'
+
+export function loadCustomConfiguration(path='/tmp'): ConfigDefault {
+    let customConfig
+    if (existsSync(path + '/config.custom')) {
+        console.log('Loading configuration...')
+
+        customConfig = readFileSync(path +'/config.custom', 'utf8')
+        customConfig = JSON.parse(customConfig)
+    } else {
+        console.log('Creating default configuration file...')
+        const config = createDefaultConfig()
+        writeFileSync(path +'/config.custom', JSON.stringify(config) , 'utf-8')
+
+        loadCustomConfiguration(path)
```

Comment:
The `customConfig` seems to be `null` in `else` casee

Reply:
it recursive

Reply:
So it will be executed, but no result assignment will be done, so the return will still be empty

Reply:
fixed

Reply:
There should be `return loadCustomConfiguration(path)`